Widgets/Bars/HealthBar.lua

```diff
@@ -30,28 +30,39 @@ function U:UnitFrame_UpdateHealthColor(button)
     local barR, barG, barB
     local lossR, lossG, lossB
     local barA, lossA = 1, 1
+    local healthPct = button.states.healthPercent
 
     if Cell.loaded then
         barA = CellDB["appearance"]["barAlpha"]
         lossA = CellDB["appearance"]["lossAlpha"]
     end
 
+    local swapHealthAndLossColors = CUF.DB.GetColors().hostileUnits.swapHealthAndLossColors and (UnitCanAttack('player', unit) or not UnitIsFriend('player', unit))
```

Comment:
This should prolly be set as a value on the widget itself to prevent too many lookups, since this function is called often.
LGTM otherwise, I'll merge and work out the optimization part.